docs/docs/machine-setup.md

```diff
@@ -1,112 +1,64 @@
+(named like this: DC?s_v3) where the question mark is the size.
 
-To deploy an enclave using real SGX hardware you need to configure the host system, and if your hardware
-does not support DCAP attestation also get access to the [Intel Attestation Service (IAS)](ias.md). At this time,
-the host must be [Linux](machine-setup.md#distribution-support), and requires the following steps:
+=== "Azure CLI"
 
-1. Install the SGX kernel driver (not required for Linux kernel versions later than 5.11).
-2. Install the Intel platform services software.
-3. Follow the instructions at the [IAS website](https://api.portal.trustedservices.intel.com/EPID-attestation) to get
-   access to the IAS servers using a whitelisted SSL key.
+    1. You need to create the VM inside a resource group. You may use an existing resource group or create one with the
+    following command:
+    ```
+    az group create -l <LOCATION> -n <GROUP-NAME>
+    ```
 
-!!! important
-    You can *develop* enclaves on Linux, Windows, or macOS hosts. To use simulation mode in Windows and macOS, you need to install and run Docker. Intel requires that the CPU must at least support [SSE4.1](https://en.wikipedia.org/wiki/SSE4#SSE4.1). Conclave works only in mock mode on new Mac computers with Apple silicon due to a [known issue](known-issues.md)
+    2. Create the VM.
+    ```bash
+    az vm create \
+    --size Standard_DC4s_v3 \
+    --image Canonical:0001-com-ubuntu-server-focal:20_04-lts-gen2:latest \
+    --name <VM-NAME> \
+    --resource-group <GROUP-NAME>
+    ```
 
+    `VM-NAME` is the name you choose for the VM, and `GROUP-NAME` is the name of your resource group.
 
-## Hardware support
+=== "Azure Portal"
 
-The machine needs support from both the CPU and firmware. Please check with your hardware manufacture to confirm that your machine supports SGX.
+    Make sure you do the following when creating your Azure Confidential Computing VM:
 
-For some machines SGX must be explicitly enabled in the BIOS/UEFI firmware screens. For others, it can be activated
-by any root user: the Conclave host API will try to activate it for you, if possible and if run with sufficient
-permissions.
+    * Use the `Ubuntu Server 20.04 (Gen2)` image
+    * Pick a size that's got plenty of RAM. You might have to click "Change size" to find the `DC4s_v3` type
+    * Ensure that the public inbound ports are open for SSH access
 
-## Hosting providers
+    ![](images/create-dc4sv3-vm.png)
+    ![](images/create-d4sv3-vm-publicinbound.png)
 
-* Microsoft Azure offers [virtual machines with SGX hardware](https://azure.microsoft.com/en-us/solutions/confidential-compute/)
-* OVH offers [rentable SGX hardware](https://www.ovhcloud.com/en-gb/bare-metal/intel-software-guard-extensions/)
+After you have logged on to the VM:
 
-## Distribution support
+1. Check that the `enclave` device is in the `/dev/sgx/` or `/dev/sgx_*` directory.
+2. Add your user into the `sgx_prv` group to give it access to SGX.
+```sh
```

Comment:
it was `bash` elsewhere, why here is `sh`?

Reply:
Changed to bash.